bot.py

```diff
@@ -1797,16 +1814,6 @@ def main():
         )
         sys.exit(0)
 
-    # Set up discord.py internal logging
```

Comment:
Oh, apparently `LOG_DISCORD` already existed, and as an environment variable. This looks hastily done and wasn't thoroughly reviewed. Let's make it proper this time. :)